packages/web/src/scss/tools/_utilities.scss

```diff
@@ -1,9 +1,37 @@
 // Inspired by React UI and Bootstrap
+@use '@tokens' as tokens;
 @use 'sass:list';
 @use 'sass:map';
 @use 'sass:meta';
+@use 'sass:string' as sass-string;
 @use 'string';
 
+@function generate-token-map($tokens-key: null, $prefix: null, $replacement: null, $scope: null) {
+    $result: ();
+    $tokens: if(
```

Comment:
Why don't you use the get-variable-if-exists function? It seems to me that you do the same.